src/main/java/com/exasol/releasedroid/adapter/communityportal/CommunityPostRenderer.java

```diff
@@ -0,0 +1,95 @@
+package com.exasol.releasedroid.adapter.communityportal;
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+/**
+ * Renders a Community Post Body.
+ */
+public class CommunityPostRenderer {
+    /**
+     * Render an Exasol release announcement community post body in the format the portal supports.
+     * 
+     * @param projectNameAndVersion name and version of the released project
+     * @param projectDescription    project description
+     * @param changesDescription    description of the changes in the current release
+     * @param gitHubReleaseLink     link to the release on the GitHub
+     * @return rendered community post body
+     */
+    public String renderCommunityPostBody(final String projectNameAndVersion, final String projectDescription,
+            final String changesDescription, final String gitHubReleaseLink) {
+        final var builder = new StringBuilder();
+        builder.append("<h2>About the project</h2>") //
+                .append(formatParagraph(projectDescription)) //
+                .append("<h2>New release</h2>") //
+                .append(formatParagraph(changesDescription)) //
+                .append("<p>For more information check out the <a href=\"") //
+                .append(gitHubReleaseLink) //
+                .append("\" target=\"_blank\" rel=\"noopener\">") //
+                .append(projectNameAndVersion) //
+                .append("</a> release on GitHub.</p>");
+        return builder.toString();
+    }
+
+    private String formatParagraph(final String projectDescription) {
```

Comment:
So you are converting md to html here. I'm wondering if it would be better to use an existing library for that.
That would for example also handle lists line breaks and so on.

Reply:
I'll check if something fitting here exist

Reply:
Switched to a library